pkgs/development/tools/mdfmt/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "mdfmt";
+  version = "29e3d55";
```

Comment:
This tool hasn't been maintained for almost 4 years. This can become a problem if vulnerabilities arises or build issues with new Golang releases.

```suggestion
  version = "unstable-20181128";
```

Reply:
Thanks for the review, and you're right but, well, there is not much to do anymore, it's just a markdown linter.
I understand the concern howerver and would understand if the PR gets rejected.